x11-wm/fvwm3/fvwm3-1.0.4.ebuild

```diff
@@ -0,0 +1,195 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8,9,10,11} )
+GO_OPTIONAL=1
+inherit autotools desktop flag-o-matic go-module python-single-r1
+
+DESCRIPTION="A multiple large virtual desktop window manager derived from fvwm"
+HOMEPAGE="http://www.fvwm.org/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/fvwmorg/fvwm3.git"
+	EGIT_BRANCH="master"
+else
+	SRC_URI="https://github.com/fvwmorg/fvwm3/releases/download/${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+SRC_URI+=" https://gitlab.com/Matt.Jolly/fvwm3-go-deps/-/raw/${PV}/${P}-deps.tar.xz?inline=false -> ${P}-deps.tar.xz"
+
+LICENSE="GPL-2 FVWM"
+SLOT="0"
+
+IUSE="bidi debug doc go htmldoc netpbm nls perl png readline rplay stroke svg tk truetype vanilla lock x86-fbsd"
+REQUIRED_USE="
+	htmldoc? ( doc )
+	${PYTHON_REQUIRED_USE}"
+
+DOCS=( NEWS )
+
+if [[ ${PV} == 9999 ]]; then
+	DOCS+=( dev-docs/COMMANDS dev-docs/DEVELOPERS.md dev-docs/INSTALL.md dev-docs/PARSING.md dev-docs/TODO.md dev-docs/NEW-COMMANDS.md )
+fi
+
+BDEPEND="
+	app-arch/unzip
+	sys-libs/zlib
```

Comment:
Libraries which are linked against are usually for DEPEND + RDEPEND, not BDEPEND.

BDEPEND is for tools which get executed (on CBUILD) during, well, the build, so stuff like unzip, Go.